kernel/src/main/java/org/kframework/parser/inner/kernel/EarleyParser.java

```diff
@@ -587,11 +588,11 @@ public List<Scanner.Token> getWords() {
     }
 
     public List<Integer> getLines() {
-      return Arrays.stream(lines).boxed().collect(Collectors.toList());
+      return Ints.asList(lines);
```

Comment:
this needs to be wrapped in Collections.unmodifiableList because Ints.asList returns a mutable list.